types/three/src/core/Raycaster.d.ts

```diff
@@ -13,14 +13,14 @@ export interface Face {
     materialIndex: number;
 }
 
-export interface Intersection {
+export interface Intersection<TIntersected extends Object3D> {
```

Comment:
can this default to = Object3D also? This change looks like it's going to cause a couple hundred line changes in my client code where we were using the Intersection type from three?

Reply:
Yeah, that sounds helpful, feel free to make a PR on [three-ts-types](https://github.com/three-types/three-ts-types).